website/src/pages/contributing/sponsors/index.tsx

```diff
@@ -18,10 +18,12 @@ const Index = () => <Layout
       </p>
     </div>
     <div className={'card sponsor'}>
-      <img
-        src={bjoernbuettner}
-        alt={'Bjoern Büttner\'s Backing'}
-      />
+      <div className={'sponsor-image'}>
+        <img
+          src={bjoernbuettner}
+          alt={'Bjoern Büttner\'s Backing'}
```

Comment:
not quite sure why there is an `oe`, could you replace it with `ö`?

Reply:
Sure! Already pushed